<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRepositoryBase</class>
 <widget class="QWidget" name="EditRepositoryBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="repoNameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="repoNamespaceLabel">
        <property name="text">
         <string>Namespace:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="repoLicenseLabel">
        <property name="text">
         <string>&amp;License:</string>
        </property>
        <property name="buddy">
         <cstring>repoLicenseEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="repoLicenseEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="repoAuthorsLabel">
        <property name="text">
         <string>&amp;Authors:</string>
        </property>
        <property name="buddy">
         <cstring>repoAuthorsEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KLineEdit" name="repoAuthorsEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="repoFileTypesLabel">
        <property name="text">
         <string>&amp;File types:</string>
        </property>
        <property name="buddy">
         <cstring>repoFileTypesList</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="repoFileTypesListLabel">
        <property name="text">
         <string notr="true">list of selected filetypes</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QListWidget" name="repoFileTypesList"/>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="repoNameEdit">
        <property name="placeholderText">
         <string>A short name for the repository</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="repoNamespaceEdit">
        <property name="placeholderText">
         <string>Prefix to show in code completion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
